charts/airbyte-worker/templates/hpa.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if eq .Values.hpa.enabled true }}
```

Comment:
Was this already here? I don't think we need a HPA in OSS. Or this here because of Cloud?

Reply:
Remove it, my bad, sorry.